cgi-bin/LJ/CleanHTML.pm

```diff
@@ -433,9 +433,12 @@ sub clean
 
             # force this specific instance of the tag to be allowed (for conditional)
             my $force_allow = 0;
+
             if (defined $action{$tag} and $action{$tag} eq "conditional") {
                 if ( $tag eq "iframe" ) {
-                    $force_allow = LJ::Hooks::run_hook( 'allow_iframe_embeds', $attr->{src} );
+                    my $can_https;
+                    ( $force_allow, $can_https ) = LJ::Hooks::run_hook( 'allow_iframe_embeds', $attr->{src} );
+                    $attr->{src} =~ s!^https?:!! if $opts->{force_https_embed};  # convert to protocol-relative URL
```

Comment:
Oops, yes.

Pushed an update. Also fixed the return value (of can_https) for those
embeds that needed special handling in EmbedWhitelist (e.g., wikimedia)